boards/zeroone/x6/rover.px4board

```diff
@@ -0,0 +1,17 @@
+CONFIG_MODULES_AIRSPEED_SELECTOR=n
```

Comment:
We carry a lot of extras in px4_fmu-v6x for testing, you can probably delete this file.

Reply:
ok,I have deleted this file.